### DISCUSSION TITLE
How to have two revealjs with different scss on the same website

### DISCUSSION THREAD

**Discussion by PHBern-RConrardy** [quarto-dev/quarto-cli#11266] [Q&A] [answered]
### Goal
I have one Quarto website, including revealjs slideshows for different employers. They need different styles (scss, or themes). I run into the same problem, no matter how I tackle the problem. The revealjs-template/scss I last rendered is used for every revealjs on the website.

### Probable cause:
When I check the css file of the rendered revealjs, it's all compiled into one css file under docs/site_libs/revealjs/dist/theme/quarto.css. 

### Desired behaviour:
There should be different css files depending on which template was chosen.

### Failed solution ideas:
1. I used extensions (one file is format: revealjs the other is format: clean-revealjs)
2. I used themes in the files
3. I used css: "filepath" stylesheet in every file

Could it be, that this usecase has not been foreseen?

### Examples
My github repo for the website is here: https://github.com/PHBern-RConrardy/lerngelegenheiten
This revealJS should be blue and yellow: https://phbern-rconrardy.github.io/lerngelegenheiten/tagungsunterlagen/2024_11_21_Zukunftsforum.html#/title-slide
This revealJS should be black and red: https://phbern-rconrardy.github.io/lerngelegenheiten/tagungsunterlagen/2024_05_07_CAS_BKB_MA.html#/title-slide

### Disclaimer
I'm not a programmer, I don't know much about Lua-filters etc. 😅

**Comment by cscheid** [comment, marked as answer]
@PHBern-RConrardy Quarto 1.6 should support this out of the box.

**Comment by PHBern-RConrardy** [comment]
thank you very much for the quick, concise answer. That solved this problem.
Have a great rest of the week.
